### PR TITLE
Add phrase management UI

### LINKED ISSUE #11: Add Phrase management UI
I seeded the DB with several phrases, but having UI around adding, removing, and updating phrases could be nice.

- We need to mention that `#GAME#` is required wherever you want the game name to show up
- Set `has_token` based on whether or not `#GAME#` is present in the string

### PR DESCRIPTION
**Pull request by ericworkman**
@Serneum I've added the scaffolding around phrases and modified it to meet the criteria in #11

- [X] We need to mention that #GAME# is required wherever you want the game name to show up
- [X] Set has_token based on whether or not #GAME# is present in the string

Listing phrases
screen shot 2016-10-16 at 1 57 08 pm

Creating a new phrase
screen shot 2016-10-16 at 1 57 17 pm

Navigation to phrases
screen shot 2016-10-16 at 1 56 58 pm

Closes #11

### CODE REVIEW

File: web/templates/phrase/index.html.eex [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <th>Has token</th>
+
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+<%= for phrase <- @phrases do %>
+    <tr>
+      <td><%= phrase.message %></td>
+      <td><%= phrase.has_token %></td>
```
Comment (Serneum) [inline web/templates/phrase/index.html.eex]: Do you have an opinion on the usefulness of showing the value of this field?

File: web/templates/phrase/show.html.eex [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<ul>
+
+  <li>
+    <strong>Message:</strong>
+    <%= @phrase.message %>
+  </li>
+
+  <li>
+    <strong>Has token:</strong>
+    <%= @phrase.has_token %>
```
Comment (Serneum) [inline web/templates/phrase/show.html.eex]: Same question here

File: web/templates/phrase/index.html.eex (hunk 1 above)
Comment (ericworkman) [inline web/templates/phrase/index.html.eex]: I really wasn't sure. I kept it just to be complete. Want me to remove them?

Comment (Serneum) [inline web/templates/phrase/index.html.eex]: Sure. I mostly had that field for internal use

File: web/templates/phrase/show.html.eex (hunk 2 above)
Comment (Serneum) [inline web/templates/phrase/show.html.eex]: Was it needed for the edit page? I remember @daveshah and I hitting some issues when removing some pages. If there's no need for it, I'd remove it

**Pull request merged**